app/src/androidTest/java/com/github/sdpcoachme/database/MockDatabase.kt

```diff
@@ -149,7 +156,7 @@ open class MockDatabase: Database {
     override fun markMessagesAsRead(chatId: String, email: String): CompletableFuture<Void> {
```

Comment:
This could use the method markOtherUsersMessagesAsRead (that would be located in Chat given other comment) to avoid code duplication.

Reply:
great catch, I overlooked that